mtgjson5/providers/tcgplayer.py

```diff
@@ -214,3 +335,26 @@ def get_tcgplayer_prices_map(
             prices_map[key].sell_foil = card_price
 
     return prices_map
+
+
+def generate_mtgjson_to_tcgplayer_map(
+    all_printings_path: pathlib.Path,
+) -> Dict[str, str]:
+    """
+    Generate a TCGPlayerID -> MTGJSON UUID map that can be used
+    across the system.
+    :param all_printings_path: Path to JSON compiled version
+    :return: Map of TCGPlayerID -> MTGJSON UUID
+    """
+    with all_printings_path.expanduser().open(encoding="utf-8") as f:
+        file_contents = json.load(f).get("data", {})
+
+    dump_map: Dict[str, str] = {}
+    for value in file_contents.values():
+        for card in value.get("cards", []) + value.get("tokens", []):
```

Comment:
I swear I wrote a function to redux this change. Might have been on your other PR.